javatests/arcs/showcase/instant/Calendar.kt

```diff
@@ -0,0 +1,40 @@
+@file:Suppress("EXPERIMENTAL_FEATURE_WARNING", "EXPERIMENTAL_API_USAGE")
+
+package arcs.showcase.instant
+
+import arcs.jvm.host.TargetHost
+import arcs.sdk.ArcsDuration
+import arcs.sdk.ArcsInstant
+import arcs.showcase.ShowcaseHost
+
+// LocalDate date = LocalDate.parse("9999-12-31");
+// ArcsInstant instant = date.atStartOfDay(ZoneId.of("Europe/Paris")).toInstant();
+
+typealias Event = AbstractCalendar.Event
+
+@TargetHost(ShowcaseHost::class)
+class Calendar : AbstractCalendar() {
+    override fun onReady() {
+        handles.events.store(
+            Event(
+                name = "Launch",
+                start = ArcsInstant.ofEpochMilli(819007320000),
```

Comment:
A comment for what this value is? Alternatively an `ArcsInstant.parse(textRepresentation)`?

Reply:
Added comments with the dates. ArcsInstant doesn't currently have a .parse. I'll add a jvm impl for it next then start on Duration.